packages/standalone/capabilities/vendir.lock.yml

```diff
@@ -1,11 +1,15 @@
 apiVersion: vendir.k14s.io/v1alpha1
 directories:
 - contents:
-  - manual: {}
+  - git:
+      commitTitle: Updating version tag used for BOM retrieval to v0.20.0 (#1951)...
+      sha: 66368ce04abcb92be48d439b09b39722a45a727b
```

Comment:
Not blocking. Just a question: does this sha get updated every time a new version of Tanzu Framework is released? Or is it updated when this component is upgraded?

Reply:
we need to manually update the sha in [vendir.yml](https://github.com/vmware-tanzu/tanzu-framework/pull/2257/files#diff-bf10adcc40972f9e0890ed9bdd733d4f0d1d08b883baf03491e512ffec9a20c8R10) whenever we make a change to capabilities CRD, right now I pulled the sha of the latest tanzu-framework release commit